appveyor.yml

```diff
@@ -3,6 +3,9 @@ build: off
 cache:
   - 'C:\\tmp'
 environment:
+  global:
```

Comment:
Please remove this, I used an environment variable in Travis and it seems to be working so far.